wagtail/wagtailimages/tests/tests.py

```diff
@@ -81,6 +81,11 @@ def test_image_tag_with_filters(self):
         self.assertTrue('width="400"' in result)
         self.assertTrue('height="300"' in result)
 
+    def test_image_tag_with_chained_filters(self):
+        result = self.render_image_tag(self.image, 'fill-200x200|height-150')
```

Comment:
Aha, it's not particularly fishy - it's just sneaking through the tag parsing code as a single unit, and when we do `'|'.join(filter_specs)` at the end, it's ending up as the same result that we would have got if they were space-separated.

Perhaps we should change the internal filter strings to space-separated too? That way, we avoid the possibility of the separator character being 'injected' in this way (and also means that an individual filter spec can safely include a `|` character without making the final chained string ambiguous, if we ever want to do that).